src/frontend/main.go

```diff
@@ -186,7 +224,24 @@ func initTracing(log logrus.FieldLogger) {
 	// In a production environment or high QPS setup please use
 	// trace.ProbabilitySampler set at the desired probability.
 	trace.ApplyConfig(trace.Config{DefaultSampler: trace.AlwaysSample()})
-	initStackdriverTracing(log)
+	initOpenCensus(log)
```

Comment:
are you making a recursive call here?

Reply:
no - `main` calls `initTracing` which calls `initOpenCensus`